[
  {
    "round_number": 1,
    "round_set": "local",
    "round_result": {
      "error": {
        "exception": "TimeoutExpired",
        "message": "Call to simulator timed out after 30 seconds.",
        "trace": "Traceback (most recent call last):\n  File \"/home/duarte/.pyenv/versions/qchack/bin/netqasm\", line 8, in <module>\n    sys.exit(cli())\n  File \"/home/duarte/.pyenv/versions/3.9.9/envs/qchack/lib/python3.9/site-packages/click/core.py\", line 1134, in __call__\n    return self.main(*args, **kwargs)\n  File \"/home/duarte/.pyenv/versions/3.9.9/envs/qchack/lib/python3.9/site-packages/click/core.py\", line 1059, in main\n    rv = self.invoke(ctx)\n  File \"/home/duarte/.pyenv/versions/3.9.9/envs/qchack/lib/python3.9/site-packages/click/core.py\", line 1665, in invoke\n    return _process_result(sub_ctx.command.invoke(sub_ctx))\n  File \"/home/duarte/.pyenv/versions/3.9.9/envs/qchack/lib/python3.9/site-packages/click/core.py\", line 1401, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n  File \"/home/duarte/.pyenv/versions/3.9.9/envs/qchack/lib/python3.9/site-packages/click/core.py\", line 767, in invoke\n    return __callback(*args, **kwargs)\n  File \"/home/duarte/.pyenv/versions/3.9.9/envs/qchack/lib/python3.9/site-packages/netqasm/runtime/cli.py\", line 346, in simulate\n    simulate_application(\n  File \"/home/duarte/.pyenv/versions/3.9.9/envs/qchack/lib/python3.9/site-packages/squidasm/run/multithread/simulate.py\", line 89, in simulate_application\n    result = mgr.run_app(app_instance, use_app_config=use_app_config)\n  File \"/home/duarte/.pyenv/versions/3.9.9/envs/qchack/lib/python3.9/site-packages/squidasm/run/multithread/runtime_mgr.py\", line 204, in run_app\n    results[name] = future.get()\n  File \"/home/duarte/.pyenv/versions/3.9.9/lib/python3.9/multiprocessing/pool.py\", line 771, in get\n    raise self._value\n  File \"/home/duarte/.pyenv/versions/3.9.9/lib/python3.9/multiprocessing/pool.py\", line 125, in worker\n    result = (True, func(*args, **kwds))\n  File \"/mnt/c/Users/duart/projects/QuTech/QCHACK2022/exp/input/app_bob.py\", line 197, in main\n    mismatch_fraction = 1 - sum(matches) / len(matches)\nZeroDivisionError: division by zero\n"
      }
    },
    "instructions": [],
    "cumulative_result": {}
  }
]